plugin/ui.tsx

```diff
@@ -275,6 +282,7 @@ class App extends SafeComponent {
   editorScriptAdded = false;
   dataToPost: any;
 
+  // TODO: THIS IS UNUSED
   async getImageUrl(
```

Comment:
Curious what this function was intended for: it calls `/upload` to upload images, but is never actually called

Reply:
ah - previous experiment that instead moved elsewhere. can be removed if we like